utils/config-utils.js

```diff
@@ -1,10 +1,30 @@
-// `CUTTLE_ENV` is needed to test prod specific files without actually putting the app in to
-// a production state, which requires an actual database connection
-const isProd =
-  process.env.CUTTLE_ENV !== undefined
-    ? process.env.CUTTLE_ENV === 'production'
-    : process.env.NODE_ENV === 'production';
-
-module.exports = {
-  isProd,
-};
+import devtools from '@vue/devtools'
+
+import { version } from '_/package.json';
+
+export function initCuttleGlobals(app) {
+  if (!window) {
+    console.error('No window object found');
+  }
+
+  const test = window.Cypress != null;
+  const cuttle = {
+    version,
+    // Expose app for debugging/testing in lower envs
+    app: window.Cypress ? app : null,
+    test,
+  };
+
+  window.cuttle = cuttle;
+
+  // Connect the devtools -- non-prod only
+  if (!import.meta.env.PROD) {
```

Comment:
If I'm the weirdo for wanting these open, we could use another env var for whether to connect devtools that I can just leave on for myself. Think the default should be off?